src/screens/ObservationItem/sections/Toolbar.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
it's better to modify app's navigation header rather than hide it and show custom one.
Here you can find how to add buttons in navigation header: https://reactnavigation.org/docs/en/header-buttons.html

Reply:
@AlexeyKaryka edited